topics/wordplate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+topic: wordplate
+created_by: Vincent Klaiber
+display_name: WordPlate
+github_url: https://github.com/wordplate
+logo: wordplate.png
+released: Mars 18, 2015
+short_description: WordPlate is a modern WordPress stack built with Composer.
+url: https://wordplate.github.io/
+---
+WordPlate is a modern WordPress stack built with Composer. It simplifies the fuzziness around WordPress development.
```

Comment:
I understand that this is the official tagline, but could you expand the last sentence a bit? What is the fuzziness?

> It simplifies the fuzziness around WordPress development by...

Reply:
What about removing the sentence all together?

Reply:
👍 to removing; would be good to replace with another sentence though, to expand on description a bit!

Reply:
Done!